lib/config/configuration.js

```diff
@@ -453,7 +453,7 @@ Configuration.prototype._processConfig = function(config) {
     }
 
     if (options.hasOwnProperty('maxErrors')) {
-        this._loadMaxError(options.maxErrors);
+        this._loadMaxError(options.maxErrors === 'null' ? null : options.maxErrors);
```

Comment:
Well, it logic to have `null` value in the `.jscsrc` config, since this is how we disable rules, consistency and all that, but passing such value to the cli seems pretty uncommon, just like passing `Infinity` if you think about it.

So i think we can keep both, but document only `null` value for `maxErrors` of `.jscsrc` (when in reality it would accept both) and document `-1` for `max-errors` of cli option  (when in reality it would again accept both) ?

Does it sounds like a good compromise?